pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
     "s3fs>=2023.9.0",
     "scikit-image!=0.23.0",
     "xarray>=0.16.1",
-    "zarr>=3.1.0,<3.1.2",
+    "zarr>=3.1.0",
```

Comment:
do we need to explicitly avoid 3.1.2, or did that issue get resolved?
And are we looking for 3.13, or 3.1.3?

Reply:
no 3.12+ is good now! The issue was that the changed around the way they accepted kwargs for storage options. we just needed to use their new contract.